### ISSUE TITLE
Create a single commit for model+client update

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#3472]
### Is your feature request related to a problem? Please describe.
The commits are different for updating models and generated code in automated releases.

Example with reference to release https://github.com/aws/aws-sdk-js-v3/releases/tag/v3.56.0
* Commit https://github.com/aws/aws-sdk-js-v3/commit/c093315d7b1f48fee007d6033f5ab399f7a9f6a2 updates models for three clients.
* A separate commit is created for updating codegen for each client:
  * config-service https://github.com/aws/aws-sdk-js-v3/commit/da332519e1e5aa13a90b2c10371780da0733e21e
  * lambda https://github.com/aws/aws-sdk-js-v3/commit/d77b86b6fa2e3bd9c9226f107a6a476dd16a7d81
  * transcribe https://github.com/aws/aws-sdk-js-v3/commit/8419c399cdb580fa608d4df09f588fe67664ab47

This will be an operational issue as there are separate commits for models update and client update.

### Describe the solution you'd like
Create a single commit for model+client update.

**Comment by AllanZhengYP** [comment]
Currently we have to create separate commits for each feature in order to generate the changelog for each client correctly. According to the [Conventional Changelog](https://github.com/conventional-changelog/conventional-changelog), the change log for each changed client is generated from the message of the commits that touched the client folder. So we have to have separate commit message in order to get separate change log entry.

I agree this is not ideal. In the future, we can investigate the [changeset](https://github.com/changesets/changesets), which offers the ability to associate multiple "features" in a single commit.

**Comment by trivikr** [comment]
> Currently we have to create separate commits for each feature in order to generate the changelog for each client correctly.

The suggestion is to add model update along with codegen updates, and not for having a single commit.
Can updates in `lambda.json` be added along with codegen changes for lambda in https://github.com/aws/aws-sdk-js-v3/commit/d77b86b6fa2e3bd9c9226f107a6a476dd16a7d81?

**Comment by AllanZhengYP** [comment]
Oh, I misunderstood the issue here. Yes, it's possible! We should take this up when we have time 😅

**Comment by AllanZhengYP** [comment]
This is fixed in 3.59.0 release. Now we only commit the feature, including the associated model changes. 

```console
commit 6bc6221428f4ecd2dcad80e919a85c907d37d487 (HEAD -> main, upstream/main)
Author: awstools <awstools@amazon.com>
Date:   Tue Mar 29 18:46:26 2022 +0000

    Publish v3.59.0

commit be368b035fc95aeba02ded15ce4ddfe02e22d488
Author: awstools <awstools@amazon.com>
Date:   Tue Mar 29 18:32:03 2022 +0000

    feat(client-organizations): This release provides the new CloseAccount API that enables principals in the management account to close any member account within an organization.

commit d0be21b8abccc03e8eb91b7561ce0b28476f02ce (tag: v3.58.0)
Author: awstools <awstools@amazon.com>
Date:   Mon Mar 28 18:45:58 2022 +0000

    Publish v3.58.0
```
